portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,7 @@
+/**
+  * This is the main javascript file for the portfolio page. Contains a showProjectDetail method.
+  * @author: shradha-khapra
+  */
 // Copyright 2019 Google LLC
```

Comment:
One line gap between doc comments and single line comments.

OR

(Better): Move the single line comments to the same doc comment.